src/ol/source/tileimagesource.js

```diff
@@ -191,3 +198,139 @@ ol.source.TileImage.prototype.useTile = function(z, x, y) {
     this.tileCache.get(tileCoordKey);
   }
 };
+
+
+/**
+ * start reloading the visible tiles, using the out-of-band mechanism
+ *
+ * @param {ol.Map} map A map instance.
+ * @api
+ */
+ol.source.TileImage.prototype.reloadVisibleTilesOutOfBand = function(map) {
+
+  // find all the visible tiles
+  var size = map.getSize() || null;
+  var viewState = map.getView().getState();
+  var extent = ol.extent.getForViewAndSize(
+      viewState.center,
+      viewState.resolution,
+      viewState.rotation,
+      size);
+  var z = this.tileGrid.getZForResolution(viewState.resolution);
+  var tileResolution = this.tileGrid.getResolution(z);
+  var tileRange = this.tileGrid.getTileRangeForExtentAndResolution(extent,
+      tileResolution);
+
+  // request reloads for all the visible tiles
+  for (var x = tileRange.minX; x <= tileRange.maxX; ++x) {
+    for (var y = tileRange.minY; y <= tileRange.maxY; ++y) {
+      var tileCoordKey = this.getKeyZXY(z, x, y);
+      this.loadRequests_[tileCoordKey] = { z: z, x: x, y: y };
+    }
+  }
+
+  // execute the load requests after a little bit
+  // this lets multiple load requests in rapid succession overwrite each other
+  // it means we end up skipping some load requests entirely if they were
+  // immediately succeeded by another request for the same tile
+  // if this function is called by a UI slider (which can generate a huge
+  // number of requests in a short time), this delay can offer a significant
+  // speedup
+  var afterDelay = function() {
+    this.executeLoadRequests(map, z, tileRange, viewState.projection);
+  };
+  window.setTimeout(goog.bind(afterDelay, this), 100);
```

Comment:
Instead of simply delaying the call, do you instead want to throttle or debounce the calls?